### PR TITLE
Adding support for displaying messages regarding rise-storage events

### PR DESCRIPTION
**Pull request by stulees**
- Displaying a message when `rise-storage` is fetching folder files
- Displaying a message when a selected storage folder has no images
- Displaying a message when a selected storage folder doesn't exist
- Applicable test coverage
- Fixed failing tests in `widget-folder-image-slider.html`. Due to changes in web component tester, the spy in "initialize" test needed to be set earlier

### CODE REVIEW

**Comment by stulees** [comment]
@alex-deaconu This PR is for the work on this card - https://goo.gl/W9SLfQ . Please review, thanks!

**Comment by alex-deaconu** [comment]
@stulees LG :+1: . Thanks.

**Comment by stulees** [comment]
@alex-deaconu Last two commits above need reviewing, should be it for this PR. Thanks!

File: src/widget/message.js [hunk 1]
```diff
@@ -0,0 +1,63 @@
+var RiseVision = RiseVision || {};
```
Comment (alex-deaconu) [inline src/widget/message.js]: @stulees this file should probably be gone, since the functionality was moved, no? Thanks.

**Comment by alex-deaconu** [comment]
@stulees from what I can tell there's a file you didn't remove in the PR. Otherwise, LG :+1: . Thanks.

**Comment by stulees** [comment]
@alex-deaconu Good catch, thanks! Have now removed the file and rebased that last commit.

**Comment by alex-deaconu** [comment]
@stulees LG :+1: . Sorry missed this yesterday. Thanks.

**Pull request merged**